src/features/moviesSlice.js

```diff
@@ -21,14 +21,14 @@ const moviesSlice = createSlice({
         setLastPage: (state, { payload: totalPages }) => {
             state.page = totalPages;
         },
-        fetchPopularMovies: state => {
+        fetchPopular: state => {
```

Comment:
In 24 code line name was changed with fetchPopularMovies to fetchPopular. In 45 code line is declare again fetchPopularMovies.

Reply:
Fixed ^-^